tests/profiler/user_workflow_fixtures.py

```diff
@@ -0,0 +1,194 @@
+from typing import List
+
+import pytest
+
+from great_expectations.core import ExpectationConfiguration, ExpectationSuite
+
+# TODO: Move these fixtures to integration tests
+
+@pytest.fixture(scope="module")
+def alice_columnar_table_single_batch():
+    profiler_config = """
+variables:
+  max_user_id: 999999999999
+  min_timestamp: 2004-10-19 10:23:54
+rules:
+  my_rule_for_user_ids:
+    domain_builder:
+      class_name: SimpleSemanticTypeColumnDomainBuilder
+      semantic_types:
+        - user_id
+    parameter_builders:
+      - name: my_min_user_id
+        class_name: MetricParameterBuilder
+        metric_name: column.min
+    expectation_configuration_builders:
+      - expectation: expect_column_values_to_be_between
+        min_value: $my_min_user_id
+        max_value: $max_user_id
+      - expectation: expect_column_values_to_not_be_null
+      - expectation: expect_column_values_to_be_of_type
+        type: INTEGER
```

Comment:
This `type` is odd.  It is indented -- what is it for?..

Reply:
This is a keyword arg for `expect_column_values_to_be_of_type` and great catch! My mistake it is actually `type_` - will fix shortly

Reply:
Fixed!

Reply:
🙇